docs/content/guides/developer/app-examples/coin-flip.mdx

```diff
@@ -433,208 +453,297 @@ Now take a look at the main function in this game, `finish_game`:
+## Frontend {#frontend}
 
-The package should successfully deploy. Now, it's time to create a frontend that can interact with it.
-
-## Frontend
+In this final part of the app example, you build a frontend (UI) that allows end users to place bets and take profits, and let's the admin manage the house.
 
 :::info
 
-The full source code for the frontend is available at the [Satoshi Coin Flip Frontend Example repository](https://github.com/sui-foundation/satoshi-coin-flip-frontend-example).
+To skip building the frontend and test out your newly deployed package, use the provided [Satoshi Coin Flip Frontend Example repository](https://github.com/sui-foundation/satoshi-coin-flip-frontend-example) and follow the instructions in the example's `README.md` file
 
 :::
 
-To expose the backend you have created to your users, you need a frontend (UI). In this section, you create a React frontend project using the [Sui Typescript SDK](https://sdk.mystenlabs.com/typescript) and the [Sui dApp Kit](https://sdk.mystenlabs.com/dapp-kit) that interacts with the deployed smart contracts.
+### Prerequisites
 
-### Initialize the project
+Before getting started, make sure you have:
 
-:::info
+- [Deployed the complete `satoshi_flip` Move package](#smart-contracts) and understand its design.
+- Installed [`pnpm`](https://pnpm.io/installation) or [`yarn`](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) to use as the package manager.
 
-The following instructions are using `pnpm` as the package manager. Follow the [`pnpm` install instructions](https://pnpm.io/installation), if needed.
+:::tip Additional resources
 
-:::
+- Tooling: [Sui Typescript SDK](https://sdk.mystenlabs.com/typescript) for basic usage on how to interact with Sui using Typescript.
+- Tooling: [Sui dApp Kit](https://sdk.mystenlabs.com/dapp-kit) to learn basic building blocks for developing a dApp in the Sui ecosystem with React.js.
+- Tooling: [`@mysten/dapp`](https://sdk.mystenlabs.com/dapp-kit/create-dapp), used within this project to quickly scaffold a React-based Sui dApp.
 
-First, initialize your frontend project. To do this rapidly, use the [`create-dapp` tool](https://sdk.mystenlabs.com/dapp-kit/create-dapp) to bootstrap the project using [dApp Kit](https://sdk.mystenlabs.com/dapp-kit). Run the following command in your terminal or console:
-
-```
-pnpm create @mysten/dapp
-```
+:::
 
-This CLI command prompts you through a couple of steps:
+### Overview
 
-1. It asks you the starter template that you want to use. Currently, there are two variants:
+The user interface (UI) of this frontend example demonstrates how to use the dApp Kit instead of serving as a production-grade product, so the Player and the House features are in the same UI to simplify the process. In a production solution, your frontend would only contain functionality dedicated to the Player, with a backend service carrying out the interactions with House functions in the smart contracts.
```

Comment:
```suggestion
The UI of this example demonstrates how to use the dApp Kit instead of serving as a production-grade product, so the Player and the House features are in the same UI to simplify the process. In a production solution, your frontend would only contain functionality dedicated to the Player, with a backend service carrying out the interactions with House functions in the smart contracts.
```
Line 686 already defines UI as replacing "frontend". I think the anagram is ubiquitous enough to the audience where that should suffice so not needed here. Other option is to remove "(UI)" from 686 and keep this as is.